buildenv/jenkins/openjdk_tests

```diff
@@ -192,6 +193,10 @@ timestamps{
                 LABEL += "&&!sw.os.ubuntu.22"
             }
 
+            if (SPEC.equals("linux_riscv64") && params.CLOUD_PROVIDER.equals("local")) {
+                LABEL = LABEL.minus("&&hw.arch.riscv")
```

Comment:
Confused. This settings will make the dynamic agent has high priority than the hardware one if both dynamic and hardware are available?

Reply:
> Confused. This settings will make the dynamic agent has high priority than the hardware one if both dynamic and hardware are available?

The hardware machine will be selected when required_docker is false. The dynamic machine will be selected when required_docker is false, add a value such as 'fyre' in DynamicAgents instead of 'local' and set CLOUD_PROVIDER to be 'fyre'. Docker will be selected when required_docker is true or CLOUD_PROVIDER is 'local'. It will select hardware one when both are available and not all hardware machines are busy if we set parameters as above.